tests/wp-unit/tests/include/Core/Posts/Metabox/DeletePostsByStatusMetaboxTest.php

```diff
@@ -0,0 +1,246 @@
+<?php
+
+namespace BulkWP\BulkDelete\Core\Posts\Metabox;
+
+use BulkWP\Tests\WPCore\WPCoreUnitTestCase;
+
+/**
+ * Test Deletion of Posts by status.
+ *
+ * Tests \BulkWP\BulkDelete\Core\Posts\Metabox\DeletePostsByStatusMetabox
+ *
+ * @since 6.0.0
+ */
+class DeletePostsByStatusMetaboxTest extends WPCoreUnitTestCase {
+
+	/**
+	 * The metabox that is getting tested.
+	 *
+	 * @var \BulkWP\BulkDelete\Core\Posts\Metabox\DeletePostsByStatusMetabox
+	 */
+	protected $metabox;
+
+	public function setUp() {
+		parent::setUp();
+
+		$this->metabox = new DeletePostsByStatusMetabox();
+	}
+
+	/**
+	 * Test Published posts can be trashed.
+	 *
+	 * @since 6.0.0
+	 *
+	 */
+	public function test_that_published_posts_can_be_trashed() {
+		$this->factory->post->create_many( 10, array(
+			'post_type' => 'post',
+		) );
+
+		$published_post = $this->get_posts_by_status();
```

Comment:
Since this variable is going to have more than one posts, let's use the plural form `$published_posts`. Change it in all the test cases.